app/controllers/api/users.php

```diff
@@ -1351,6 +1351,17 @@ function createUser(string $hash, mixed $hashOptions, string $userId, ?string $e
 
         $user = $dbForProject->updateDocument('users', $user->getId(), $user);
 
+        $sessions = $user->getAttribute('sessions', []);
+        $onPasswordChange = $project->getAttribute('auths', [])['onPasswordChange'];
```

Comment:
Let's call this `invalidate` and set to `false` if it's not in the auths array, so that existing projects keep the same behaviour until they enable it